dev/TabView/TabView.cpp

```diff
@@ -341,6 +345,18 @@ void TabView::OnListViewLoaded(const winrt::IInspectable&, const winrt::RoutedEv
     }
 }
 
+void TabView::OnTabStripPointerExited(const winrt::IInspectable& sender, const winrt::PointerRoutedEventArgs& args)
+{
+    if (updateTabWidthOnPointerLeave)
+    {
+        UpdateTabWidths();
+        auto scopeGuard = gsl::finally([this]()
+        {
+            updateTabWidthOnPointerLeave = false;
+        });
```

Comment:
Scope guarded flag should be declared before UpdateTabWidths() is called.  This is so that if UpdateTabWidths() throws but that exception is later handled we will still update this flag.

Reply:
Good point, moved the UpdateTabWidths call behind the scopeguard.